cmd/e2e-runner/main.go

```diff
@@ -174,6 +169,54 @@ func realMain(ctx context.Context) error {
 	return srv.ServeHTTPHandler(ctx, mux)
 }
 
+func bootstrap(ctx context.Context, db *database.Database, cfg *config.E2ERunnerConfig) mux.MiddlewareFunc {
+	logger := logging.FromContext(ctx).Named("e2erunner.bootstrap")
+	bootstrapFn := func() error {
```

Comment:
Wouldn’t this be simpler with sync.Once?

Reply:
it's not a true "once" see, the note on line 201

Reply:
I see